WebDriverAgentLib/Utilities/FBScreen.m

```diff
@@ -21,8 +21,16 @@ + (double)scale
 
 + (CGSize)statusBarSizeForApplication:(XCUIApplication *)application
 {
-  XCUIElement *mainStatusBar = application.statusBars.fb_firstMatch;
-  if (!mainStatusBar || !mainStatusBar.fb_isVisible) {
+  XCUIApplication *app = application;
+  BOOL expectVisibleBar = true;
+
+  if (@available(iOS 13.0, *)) {
+    app = [[XCUIApplication alloc] initPrivateWithPath:nil bundleID:@"com.apple.springboard"];
```

Comment:
You can use `FBSpringboardApplication.fb_springboard` by `FBSpringboardApplication.h`